requirements.txt

```diff
@@ -2,12 +2,14 @@ Django==1.10.1
 django-cors-headers==1.3.1
 django-filter==1.0.1
 djangorestframework==3.4.7
+djangorestframework-jsonapi==2.1.1
 DukeDSClient==0.2.14
 funcsigs==1.0.2
+inflection==0.3.1
+lando-messaging==0.1
 mock==2.0.0
 pbr==1.10.0
+pika==0.10.0
 PyYAML==3.12
 requests==2.11.1
 six==1.10.0
-git+git://github.com/Duke-GCB/lando-messaging.git
```

Comment:
Ah, this is a weird thing with pip. When the requirement is specified this way, it gets simplified to `lando-messaging==0.1` when freezing. Instead, we need to specify it like

```
-e git+https://github.com/Duke-GCB/lando-messaging.git@06dd93b1cde8083e4a11c96f3f3e77658932c674#egg=lando-messaging
```

A tag is probably better there